cpp/include/models/i_jni_renderer.hpp

```diff
@@ -0,0 +1,22 @@
+#ifndef _RIVE_ANDROID_JAVA_RENDERER_HPP_
+#define _RIVE_ANDROID_JAVA_RENDERER_HPP_
+
+#include "rive/renderer.hpp"
+#include <jni.h>
+
+namespace rive_android
+{
+	/**
+	 * Interface for a generic JNI Renderer.
+	 */
+	class IJNIRenderer
+	{
+	public:
+		virtual ~IJNIRenderer() {}
+		virtual rive::RenderPaint* makeRenderPaint() = 0;
+		virtual rive::RenderPath* makeRenderPath() = 0;
```

Comment:
this was supposed to mimic the interface we have in C++ to simplify the implementation of multiple renderers on Android. 
Do we need it still? If so, we might need to augment with `makeRenderImage()` here too.
Otherwise, we can remove it!